### PR TITLE
Add TensorProtos to represent a list of Tensors

### PR DESCRIPTION
**Pull request by yinghai**
Sometimes we want to serialized/deserialize a list of tensors for testing purposes (inputs/outputs). A repeated TensorProto message makes this convenient.

### CODE REVIEW

File: onnx/onnx.in.proto [hunk 1]
```diff
@@ -456,6 +456,11 @@ message TensorProto {
   repeated uint64 uint64_data = 11 [packed = true];
 }
 
+// TensorProtos stores multiple TensorProto objects in one single proto.
+message TensorProtos {
```
Comment (houseroad) [inline onnx/onnx.in.proto]: Wanna try sequence: https://github.com/onnx/onnx/blob/master/onnx/onnx.proto#L515

**Comment by houseroad** [comment]
Btw, there is no good way to represent sequence and map type weights in initializer list. We have created an issue: https://github.com/onnx/onnx/issues/2410

File: onnx/onnx.in.proto (hunk 1 above)
Comment (yinghai) [reply onnx/onnx.in.proto]: How to use this?

**Comment by yinghai** [comment]
What's the status about this?

**Pull request closed without merging**